sphinx/debugger.rst

```diff
@@ -0,0 +1,113 @@
+The Gillian Debugger
```

Comment:
Is there a rendered version of this?

Reply:
Sure, I've attached a rendered version [here](https://github.com/GillianPlatform/Gillian/files/10951864/Gillian.debugger.docs.zip).

It could be a good idea to get CI to submit built docs as an artefact?

Reply:
Not the right file 🙃 
But I'll just checkout and build myself